src/main/java/com/google/firebase/cloud/FirestoreClient.java

```diff
@@ -108,7 +108,7 @@ static Firestore getFirestore(FirebaseApp app, String database) {
    *     instance.
```

Comment:
Hey folks, were these public docs were reviewed by a tech writer before merging the changes?
@tom-andersen @wu-hui

Reply:
We involved @markarndt in multidb texts last year. The changes were repetitive across SDKs, so he didn't review all of them.

Is the process now to have tech writers mark PRs directly with approval?